core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/config/ExistingStackPatcherConfigPatch.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.job.stackpatcher.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExistingStackPatcherConfigPatch {
```

Comment:
Maybe ExitingStackPatchConfig would be better as a class name.

Reply:
It is under ExistingStackPatcherConfig's each stack patch type, so how about ExistingStackPatcherPatchTypeConfig?

Reply:
ExistingStackPatchTypeConfig? :-)